src/components/06_wrappers/Default/Default.js

```diff
@@ -38,14 +41,50 @@ const iconMap = {
 };
 
 class Default extends React.Component {
+  state = {
+    drawerOpen: false,
```

Comment:
If you transition between pages does the state of the drawer remain the same? Is this something that should be better kept within Redux?

Reply:
Good point!